android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DocumentField.java

```diff
@@ -105,19 +106,21 @@ public void createLocalFile(String path) {
 
     @Override
     protected DocumentFile convertFromString(String string) {
-        if (string == null || string.isEmpty()) {
+	    if (string == null || string.isEmpty()) {
             return null;
         }
 
-        DocumentRemoteFile result = null;
-
         try {
-            result = new DocumentRemoteFile(string);
+	        JSONObject json = new JSONObject(string);
```

Comment:
This value can be an URL, so you can't check only the `JSONObject`. Please check the DocumentRemoteFile constructor implementation, maybe your solution could be there.